modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -71,6 +71,7 @@ def eval_insert(modin_df, pandas_df, **kwargs):
         modin_df,
         pandas_df,
         operation=lambda df, **kwargs: df.insert(**kwargs),
+        __inplace__=True,
```

Comment:
[Insert function](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.insert.html?highlight=insert#pandas.DataFrame.insert) doesn't return anything, so we should explicitly use `__inplace__=True` here.